andrew_eye_stream_data.py

```diff
@@ -0,0 +1,50 @@
+from talon import app, ui
+from talon_plugins.eye_mouse import tracker
+from talon.track.geom import EyeFrame
+
+main = ui.main_screen()
+
+# Lightweight simple recording for eye tracking.
+class EyeDataStream:
+    def __init__(self):        
+      self.x_coords = []
+      self.y_coords = []
+
+    def on_gaze(self, b):
+      l, r = EyeFrame(b, 'Left'), EyeFrame(b, 'Right')
+      p = (l.gaze + r.gaze) / 2
```

Comment:
Are `l` and `r` made up of integer types or float types? I wonder if the shift left could be caused by an implicit flooring of the number from integer division.

Reply:
They are float types, but after inspecting I do not believe this is an issue

Reply:
Thanks for checking!